composer.json

```diff
@@ -27,7 +27,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
`composer.lock` needs updating too - probably a good idea to drop everything below `8.0` too.

Reply:
Done :)